src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -162,6 +162,19 @@ public async Task<FormulaValue> EvalAsync(string expressionText, CancellationTok
             return await check.Expression.EvalAsync(parameters, cancel);
         }
 
+        public DefineFunctionsResult DefineFunctions(string script)

```

Comment:
>DefineFunctions [](http://example.com/codeflow?start=37&length=15)

Can this method be static?
Ie, is it actually changing anything on Engine? Or just exposing Parse?

Reply:
This method cannot be static